includes/class-plugin-manager.php

```diff
@@ -16,6 +16,48 @@
  */
 class Plugin_Manager {
 
+	/**
+	 * Get info about all the managed plugins and their status.
+	 *
+	 * @todo Define what the structure of this looks like better and load it up from a config or something.
+	 *
+	 * @return array of plugins info.
+	 */
+	public static function get_managed_plugins() {
```

Comment:
This will need to be reconciled with `get_managed_plugins` in [the other PR](https://github.com/Automattic/newspack-plugin/pull/9/files#diff-044b9390f497ea129e19496244893c6eR26). I'll handle that once either of these PRs is merged.